### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by edavidj** [OpenLiberty/draft-guide-rest-client-reactjs#9]
- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Resolve anything that does not work
- [x] Resolve anything usability issues (confusing or hard to follow)
- [x] Achieve basic understanding/working knowledge of the topic in the guide

Comments:
- 14 minutes to work through.
- Wording is clear and did not run into any bugs while doing the guide.
- In terms of clarity, I think the last section on testing could tell the user to run the `npm run test` command from `frontend` just to show how it's actually run. Or if it's already run by dev mode at some point just an explanation would help. 
- In the axios section it's a little misleading to say you need axios (even if it's highly recommended) since you can use the `fetch()` api without additional installation in modern browsers.
- This is a little nitpicky, but it'd be a good look to adopt [React Hooks](https://reactjs.org/docs/hooks-intro.html) + functional components rather than the class based one seen in `ArtistTable.js`. This is considered the best practice since 16.8 (but not high priority). It would ultimately require changing `componentDidMount` to a [useEffect](https://reactjs.org/docs/hooks-effect.html) hook, and the state variables to [useState](https://reactjs.org/docs/hooks-state.html)

**Comment by DannyS03** [comment]
Thanks for the feedback, @edavidj.

- The suggestion for testing also came up in the SME review; I added the configuration in the **pom.xml** to run "all" the tests **(npm test a)**, where the user can add the configuration if they wish to do so (since testing is not one of the focuses of the guide). 
> In terms of clarity, I think the last section on testing could tell the user to run the `npm run test` command from `frontend` just to show how it's actually run. Or if it's already run by dev mode at some point just an explanation would help. 


- I took out the sentence that was misleading.
> In the axios section it's a little misleading to say you need axios (even if it's highly recommended) since you can use the `fetch()` api without additional installation in modern browsers.

- Oh not nitpikcy, that's a good suggestion. Most of the material I looked at for our use case were still using class components so I thought it would be ideal to follow that. I would leave it for now and if it comes up again we should surely make the switch.
> This is a little nitpicky, but it'd be a good look to adopt [React Hooks](https://reactjs.org/docs/hooks-intro.html) + functional components rather than the class based one seen in `ArtistTable.js`. This is considered the best practice since 16.8 (but not high priority). It would ultimately require changing `componentDidMount` to a [useEffect](https://reactjs.org/docs/hooks-effect.html) hook, and the state variables to [useState](https://reactjs.org/docs/hooks-state.html)

**Comment by DannyS03** [comment]
Changes verified by Ethan.
